storefront/.env.defaults

```diff
@@ -5,3 +5,4 @@ SKIP_INVALID_REMOTES=false
 UNCHAINED_ENDPOINT=http://localhost:4010/graphql
 GETCOCKPIT_ENDPOINT=http://localhost:4020
 GETCOCKPIT_TOKEN=token
+COOKIE_DOMAIN=
```

Comment:
Shouldn't it be `localhost:3000`?

Reply:
It will work when left empty and set to localhost automatically by the browser